aea/connections/base.py

```diff
@@ -45,6 +46,15 @@
 logger = logging.getLogger(__name__)
 
 
+class ConnectionStates(Enum):
```

Comment:
Ok, why do we have ConnectionStates and ConnectionStatus now? Seems like ConnectionStatus needs to be removed..

Reply:
it used by  multiplexer

Reply:
it used by  multiplexer